.github/workflows/review.yml

```diff
@@ -43,7 +43,7 @@ jobs:
         run: |
           if [ "${{ contains(github.event.pull_request.labels.*.name, 'run-ci/clang-format') }}" = "true" ]; then
             echo "run=true" >> "${GITHUB_OUTPUT}"
-          elif git diff --name-only origin/${{ github.base_ref }} HEAD | grep -Eq '.*\.\(cpp|cxx|c|hpp|hxx|h\)$' ; then
+          elif git diff --name-only origin/${{ github.base_ref }} HEAD | grep -Eq '.*\.(cpp|cxx|c|hpp|hxx|h)$' ; then
```

Comment:
```suggestion
          elif git diff --name-only origin/${{ github.base_ref }} HEAD |  grep -Eq '\.cpp$|\.cxx$|\.c$|\.hpp$|\.hxx$|\.h$' ; then
```

Reply:
Why is this better? I think it was clearer